Ray MVP docs (including new features) (#261)
golemfactory/golem-docs: src/pages/docs/creators/ray/mainnet.md

    │  token: GLM        │                │            │  requested  │  0 GLM     │  0 GLM     │           │
    └────────────────────┴────────────────┴────────────┴─────────────┴────────────┴────────────┴───────────┘
  
  You can use the Golem Onboarding portal to top up: https://glm.golem.network/#/onboarding/budget?yagnaAddress=0xae5d43485a341bc4f167f6a4fda0749f1d7f3e11&network=polygon

You don't seem to have any GLM tokens on your Golem wallet.

```

Even if you do have the required funds, `ray up` will nevertheless print the onboarding link as a reminder, in case you'd like to top up your wallet.

The onboarding process is a convenient and effortless way of providing your Ray on Golem node with funds allowing it to run on the mainnet.

> Initially, your Golem node generates and later manages its own Ethereum wallet that is used to perform transactions. This wallet starts empty and Ray on Golem can transparently obtain testnet tokens when you run it on testnet. 

Reviewer: ```suggestion
Initially, your Golem node generates and later manages its own Ethereum wallet, which is used to perform transactions. This wallet starts empty and Ray on Golem transparently obtains funds needed to operate on testnet. 
```

Golem Network is peer-to-peer, which means that providers' proposals are not always available at first. They get broadcasted only from time to time.

> This means that when you first start Ray on Golem it might take a while to find a provider matching all your requirements.

Reviewer: ```suggestion
This means that when you first start Ray on Golem, it might take a while to find a provider matching all your requirements.
```
